#==================================================================================================
# Upstream patch to fix errno-related build failure on some systems
#
# Source: https://github.com/ggerganov/llama.cpp/pull/11296
# Issue:  https://github.com/ggerganov/llama.cpp/issues/11295
#==================================================================================================
--- src/llama-mmap.cpp.orig	2025-01-18 15:49:52.000000000 -0500
+++ src/llama-mmap.cpp	2025-01-18 15:50:19.000000000 -0500
@@ -7,6 +7,7 @@
 #include <cstring>
 #include <climits>
 #include <stdexcept>
+#include <cerrno>
 
 #ifdef __has_include
     #if __has_include(<unistd.h>)
